Sources/PackageLoading/ModuleMapGenerator.swift

```diff
@@ -186,6 +186,12 @@ public struct ModuleMapGenerator {
         }
         stream <<< "    export *\n"
         stream <<< "}\n"
+        if let swiftHeaderPath = swiftHeaderPath {
```

Comment:
Why is new code needed here?  Wasn't it already supported to use Swift modules from other Objective C modules?